### PR TITLE
feat: codelens to show real value

### PR DESCRIPTION
**Pull request by nliu-ms**
show real values for manifest template (local & dev)
image

### CODE REVIEW

**Comment by nliu-ms** [comment]
E2E TEST: n/a

File: packages/vscode-extension/src/codeLensProvider.ts [hunk 1]
```diff
@@ -98,16 +116,62 @@ export class ManifestTemplateCodeLensProvider implements vscode.CodeLensProvider
...
+
+        const localValue = getPropertyByPath(localEnvInfo, key);
+        title = `${title} ${environmentManager.getLocalEnvName()}: ${localValue}`;
+
+        const defaultValue = getPropertyByPath(defaultEnvInfo, key);
+        title = `${title}, ${defaultEnvInfo.envName}: ${defaultValue}`;
+
+        lens.command = {
+          title: title,
+          command: "fx-extension.openConfigState",
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/codeLensProvider.ts:167]: Does this feature show real value for config?  If so, should pass 'config' as type here.

**Comment by HuihuiWu-Microsoft** [comment]
If we show local&default env values for codelens, will users know he can also click on the codelens to check values for other env?

**Comment by nliu-ms** [comment]
> If we show local&default env values for codelens, will users know he can also click on the codelens to check values for other env?

We will add a hover to show values for other environments.

File: packages/vscode-extension/src/codeLensProvider.ts (hunk 1 above)
Comment (nliu-ms) [reply packages/vscode-extension/src/codeLensProvider.ts:167]: Updated, thanks!

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**